criu/crtools.c

```diff
@@ -405,6 +405,9 @@ int main(int argc, char *argv[], char *envp[])
 	       "                        this requires running a second instance of criu\n"
 	       "                        in lazy-pages mode: 'criu lazy-pages -D DIR'\n"
 	       "                        --lazy-pages and lazy-pages mode require userfaultfd\n"
+		   "  --on-demand           restore pages on demand\n"
+		   "                        unlike lazy-pages using userfaultfd, on-demand use\n"
+	       "                        sys_mmap directly, so the kernel handle the page fault\n"
```

Comment:
@Yangfisher1 Could you merge this commit with the commit introducing `--on-demand` and add document this option in `Documentation/criu.txt` as well?

Reply:
Ok, I'll add document info.